ground/gcs/src/plugins/systemhealth/systemhealthgadgetwidget.cpp

```diff
@@ -138,7 +138,6 @@ SystemHealthGadgetWidget::~SystemHealthGadgetWidget()
 
 void SystemHealthGadgetWidget::setSystemFile(QString dfn)
 {
-    setBackgroundBrush(QBrush(Utils::StyleHelper::baseColor()));
```

Comment:
I gave it a try and I think I'm gonna strip it. A lot of the UI in the GCS uses system colors anyway, and there are some widgets, that are probably supposed to use it, that actually don't and use their own static colors. I ran for instance into the PFD and OPMap doing this.

This is going to take some time, because I have to remove StyleHelper and see what breaks.